cpu/esp32/Makefile.include

```diff
@@ -140,6 +140,8 @@ LINKFLAGS += -Wl,--warn-unresolved-symbols
 FLASH_MODE ?= dout  # FIX configuration, DO NOT CHANGE
 FLASH_FREQ  = 40m   # FIX configuration, DO NOT CHANGE
 FLASH_SIZE ?= 2MB
+FLASHFILE ?= $(ELFFILE)
```

Comment:
For which purpose is this define required? Is it something used by murdock?

Reply:
Yes, the "test-murdock" target needs to know which file to send to the RasPi worker nodes.

Reply:
When setting `FLASHFILE`, the flash/preflash targets should actually be using `FLASHFILE` instead of `ELFFILE`. I have two commits in https://github.com/RIOT-OS/RIOT/pull/8838 that I can PR.

Reply:
This now has its PR https://github.com/RIOT-OS/RIOT/pull/11648